components/Profile/TweetEmbed.jsx

```diff
@@ -0,0 +1,83 @@
+import { ReloadOutlined } from '@ant-design/icons';
+import { Button, Card, Spin, Typography } from 'antd';
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+import { TwitterTweetEmbed } from 'react-twitter-embed';
+import styled from 'styled-components';
+
+const { Text } = Typography;
+
+const TweetPlaceholder = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  align-items: center;
+  justify-content: center;
+  margin: 16px;
+  min-height: 120px;
+`;
+
+const StyledCard = styled(Card)`
+  width: 100%;
+`;
+
+export const TweetEmbed = ({ points, tweetId, width = 250 }) => {
+  const [isError, setIsError] = useState(false);
+  const [isMounted, setIsMounted] = useState(true);
```

Comment:
not a big issue, but naming is weird here, as `isMounted` will be `true` before it's mounted..

```suggestion
  const [isReloading, setIsReloading] = useState(true);
```

Reply:
I thought about the same name at first, makes sense to rename